Added the MongoDB Data API blog post

CoolAssPuppy opened the pull request:
## I have read the [CONTRIBUTING.md](https://github.com/supabase/supabase/blob/master/CONTRIBUTING.md) file.

YES

## What kind of change does this PR introduce?

New blog post

## What is the current behavior?

The blog post doesn't exist.

## What is the new behavior?

The blog post exists.

## Additional context

Need to get an OG image from the design team.

CoolAssPuppy committed:
Added the MongoDB Data API blog post

fsansalvadore commented on apps/www/_blog/2025-03-20-migrating-mongodb-data-api-with-supabase:
```diff
@@ -0,0 +1,155 @@
+---
+title: 'Migrating from the MongoDB Data API to Supabase'
+description: "A guide to migrating from the MongoDB Data API to Supabase."
+author: prashant_sridharan
```
```suggestion
author: prashant
```

fsansalvadore replied:
I previously added you as an author as simply `prashant` (you'll appear with full name)

https://github.com/supabase/supabase/blob/3a45d62309b98de3199778c78c8c78cf1c411fc4/apps/www/data/Authors.json#L26C3-L31C4

fsansalvadore committed:
Update apps/www/_blog/2025-03-20-migrating-mongodb-data-api-with-supabase

CoolAssPuppy committed:
Merge branch 'master' into mongodb-blog-post

fsansalvadore committed:
chore: add prashant and fix blog post

fsansalvadore committed:
chore: fix prettier

avallete committed:
chore(studio): move Query to pgMeta add tests (#34232)

* chore(studio): move Query to pgMeta add tests

- Move the Query builder from studio to pgMeta
- Add e2e tests over the generated sql to ensure syntax and runtime
  result over pg database
- fix bug with orde by for table with undefined column

* chore: fix query import path

* chore: set ES target for lint

* chore: add github action for pg-meta test package

* chore: add tsconfig to sparse checkout

ivasilov committed:
feat: Library (#34294)

* Copy the design-system app into a new one for ui-library.

* Remove unneeded content.

* Add supabase config.

* Cleanup the css.

* Add bunch of packages.

* Cleanup the registry.

* Regenerate the registry.

* Add needed components for documenting components.

* Add the pages for the components.

* Fix the RegistryBlock.

* Various fixes.

* Add a turbo definition for ui-library.

* Rename Remix to React Router.

* Reorder the pages for all frameworks.

* Remove the bottom pager.

* Fix the pages and command menu.

* Various fixes.

* Minor fixes.

* Add ai editor rules.

* Various fixes.

* Add local supabase env vars.

* Try to fix a package error.

* Bunch of various fixes.

* Fix lint errors.

SaxonF committed:
fix duplicate policy name (#34316)

* fix duplicate policy name

* style

joshenlim committed:
Update realtime settings based on feedback (#34268)

* Update realtime settings based on feedback

* Add pool size warning

* Update pool size warning threshold from 80 to 50%

* Smol update

* Check

* Check

* Fix

* Update label

tgallacher committed:
chore: improve memoization of getDiskAttributes (#34287)

* chore: improve memoization of useRemainingDurationForDiskAttributeUpdate

* Fix

---------

Co-authored-by: Joshen Lim <joshenlimek@gmail.com>

joshenlim committed:
Chore/support updating email and unlinking identities (#33966)

* Support updating email address for email identity

* Support unlinking identity

* Deprecate AccountInformation component

* Invalidate identities after unlinking

* Address feedback

* Smol

* Fix type issues

* Update toast message

* Set up email change pending badge

* Fix

* Address feedback

* Resolve ts-expect-errors

* Fix TS issues + Fix GH icon on light mode + Address feedback

xkjay committed:
Fix typo in Database docs (#34318)

Fix typo

Co-authored-by: Coder178 <88607322+Fighter178@users.noreply.github.com>

delgado3d committed:
feat: change default US region to us east (#34322)

joshenlim committed:
Authorized apps in org settings add copy action for App ID (#34321)

Authorized apps in org settings add copy action for add ip

encima committed:
move API keys launch date to Q2 2025 (#34151)

joshenlim committed:
Just to trigger redeploy (#34326)

TheOtherBrian1 committed:
Update metabase.mdx | a user complained about IPv4 issues with metabase (#34309)

* Update metabase.mdx | a user complained about IPv4 issues with metabase

* Update metabase.mdx

kevcodez committed:
feat: Dedicated Pooler rollout (#34240)

CoolAssPuppy committed:
Added Fauna blog post

charislam committed:
feat(docs): ts hover annotations on code blocks (#34327)

CoolAssPuppy committed:
Added OG and thumb images to blog posts

CoolAssPuppy committed:
Fixed a style issue

kevcodez committed:
format

CoolAssPuppy committed:
Merge branch 'master' into mongodb-blog-post
